### DISCUSSION TITLE
Image save node help

### DISCUSSION THREAD

**Discussion by mr-purr** [WASasquatch/was-node-suite-comfyui#417] [Q&A]
I want to have the checkpoint name as my image prefix but cant figure it out 
Please help

**Comment by aotillar** [comment]
I'm in this same boat. I'm stuck trying to make custom prefixes for my images as a timestamp, or as a text file output for my wildcard prompts. The best I can come up with so far is that ['time'] is a token in the nodes. I tried going into the code and editing it myself but I was unsuccessful.

**Reply by aotillar** [reply]
Please ignore - I answered my own question. Like a stupid naked ape, i did not read the readme. Thank you WAS for everything.
